pkg/utils/labels.go

```diff
@@ -117,6 +117,22 @@ func (ls *Labels) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Hash returns a hash value for the label set.
+func (ls Labels) HashWithMetricName() uint64 {
```

Comment:
try to avoid duplicating code.
you can create `func hash (ignoreFields []string)` and call it from both public methods

Reply:
FYI: latest Prometheus provide veriety of hashing finctions
https://github.com/v3io/prometheus/blob/v3io-prometheus-v2.4.3/pkg/labels/labels.go#L115

Reply:
I don't see any upsides to code sharing in this way, especially with functionality that no one will ever use for any other purpose (ignoreFields).

Reply:
What about the Prometheus code, are you suggesting I use it...?

Reply:
Not in this version. After merging 2.4.3 to the master we might take some good things